docs/guide/writing/beginners-guide-to-docs.rst

```diff
@@ -1,323 +1,210 @@
-A beginner's guide to writing documentation
-===========================================
+How to write software documentation
+====================================
 
-.. note:: This is a write up of a `Presentation <https://speakerdeck.com/ericholscher/writing-docs-a-beginners-guide-to-writing-documentation>`_ .
-          Please provide feedback to `@ericholscher`_.
-          You can view the source on `GitHub`_.
+As a software developer or engineer, there is a magical feeling that comes from releasing open-source code.
+The feeling is a mix of terror and excitement.
+You're eager to tell the world about your new project but despair at where to start. Even professional writers 
+know the mixed emotions caused by a blank page.
 
-.. _@ericholscher: http://twitter.com/ericholscher
-.. _GitHub: https://github.com/writethedocs/www/blob/master/docs/guide/writing/beginners-guide-to-docs.rst
+Good software documentation helps relieve that fear. So, have no fear! Use this guide to document your first open-source project for 
+public release. 
 
-..
-
-	| Camera pans from stage left.
-	| It shows a text editor, open to a blank page.
-	| A person hunched in front, head to desk.
+.. _why:
 
-The scene above is well known to everyone who writes for a living;
-the mixed emotions of a blank page.
-Full of excitement, fresh with a new beginning.
-Yet also full of despair, where do you even start?
+Why write software documentation?
+---------------------------------------
 
-I am here to stop this scene from playing out.
+Understand your code in 6 months
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This heading reads to me like "learn to code in six months" not "understand your own code six months from now"